### ISSUE TITLE
HEVM + EIP-3155/FuzzyVM

### ISSUE THREAD

**Issue by axic** [dapphub/dapptools#648]
It would be nice to have hevm integrated into the [FuzzyVM](https://github.com/MariusVanDerWijden/FuzzyVM) ([blog post](https://mariusvanderwijden.github.io/blog/2021/05/02/FuzzyVM/)) effort. The first step towards that would be implementing tracing output akin to [EIP-3155](https://eips.ethereum.org/EIPS/eip-3155).

**Comment by MrChico** [comment]
Agreed. There is some support for the EIP-3155 trace like functionality under the `--jsontrace` flag available under the `hevm exec` and `hevm bc-test` commands. I haven't confirmed that it is fully 3155 compliant yet.
Another missing piece here is that hevm currently runs the ethereum tests in the `BlockchainTests/GeneralStateTests` flavor, rather than the `GeneralStateTests` flavor
